### ISSUE TITLE
Using DDPG_update2.py with pendulum, reward not converging

### ISSUE THREAD

**Issue by lisayan** [MorvanZhou/Reinforcement-learning-with-tensorflow#40]
Hi, I am using https://github.com/MorvanZhou/Reinforcement-learning-with-tensorflow/blob/master/contents/9_Deep_Deterministic_Policy_Gradient_DDPG/DDPG_update2.py and running it just as given, except with MAX_EPISODES=1000 and MAX_EP_STEPS=1000.

In the beginning, it starts off with reward values ~-7000, but at the end it seems like it did not converge since there are still values of -3203, -42, -481, etc. Is this normal, or am I missing something?

**Comment by MorvanZhou** [comment]
You can try gradually lowering the learning rate so that it can converge at the end. This is caused by large learning rate.
